<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DNSNameDialog_q</class>
 <widget class="QWidget" name="DNSNameDialog_q">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DNS Name</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame7">
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame14">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Sunken</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="textLabel1">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="obj_name"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="dnsrecLabel">
             <property name="text">
              <string>DNS record:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="dnsrec">
             <property name="toolTip">
              <string>DNS 'A' record name entered in this field will be converted to an IP address using a DNS query during policy compilation if "Compile time" is selected, or during firewall policy activation if "Run time" is selected.</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="resolveLabel">
             <property name="text">
              <string>Resolve:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QRadioButton" name="r_compiletime">
             <property name="text">
              <string>Compile time</string>
             </property>
             <property name="toolTip">
              <string>Resolve the DNS name during policy compilation. The resulting IP address is embedded in the generated firewall script.</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QRadioButton" name="r_runtime">
             <property name="text">
              <string>Run time</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Resolve the DNS name when the firewall script runs on the target machine. The script will perform the DNS lookup each time it is activated.</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="CommentKeywords" name="commentKeywords" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommentKeywords</class>
   <extends>QWidget</extends>

   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>obj_name</tabstop>
  <tabstop>dnsrec</tabstop>
  <tabstop>r_compiletime</tabstop>
  <tabstop>r_runtime</tabstop>
 </tabstops>
 <resources>
  <include location="MainRes.qrc"/>
 </resources>
</ui>
